core/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -139,6 +145,19 @@ service BlockMasterClientService {
    */
   rpc GetWorkerInfoList(GetWorkerInfoListPOptions) returns (GetWorkerInfoListPResponse);
 
+  /**
+   * Returns a list of decommission workers information.
+   */
+  rpc GetDecommissionedWorkerInfoList(GetWorkerReportPOptions)
+      returns (GetWorkerInfoListPResponse);
```

Comment:
remove this RPC and reuse getWorkerReport

Reply:
Fixed.